codespell_lib/data/dictionary.txt

```diff
@@ -1488,6 +1488,7 @@ alhapetically->alphabetically
 alhapeticaly->alphabetically
 alhapets->alphabets
 alhough->although
+alhpa->alpha
```

Comment:
I suggest adding `alhpabet`, alhpabets, `alhpabetical`, `alhpabeticaly`, `alhpabetically`, if you have the time.

Reply:
Sure!

Reply:
Done, here are the corresponding hits:
https://github.com/search?q=alhpabet&type=code => 820 hits
https://github.com/search?q=alhpabetical&type=code => 195 hits
https://github.com/search?q=alhpabetically&type=code => 82 hits
https://github.com/search?q=alhpabeticaly&type=code => 16 hits
https://github.com/search?q=alhpabets&type=code => 75 hits